stock_picking_mto/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Stock Picking MTO',
```

Comment:
I'm missing a summary key here and I agree that the name is a little bit misleading